### DISCUSSION TITLE
[Enhancement] [select] Select 选择器的blur事件返回的参数添加  input输入的value

### DISCUSSION THREAD

**Discussion by OnlY0226** [element-plus/element-plus#13561] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

在 allow-create为true时，
想在 Select 选择器的blur事件将input的值直接赋给select，
但是blur事件没有返回 input的值

### Additional comments

_(empty)_

**Comment by kaine0923** [comment]
你如果要拿输入的`input`值，可以通过`filter-method`中转一下，[ElementPlus Playground](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYsIHZlcnNpb24gYXMgdnVlVmVyc2lvbiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgZXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgRWxlbWVudFBsdXMsIExvYWRpbmcgfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcblxuY29uc3QgbXNnID0gcmVmKCdIZWxsbyBXb3JsZCEnKVxuY29uc3QgdmFsdWUgPSByZWY8c3RyaW5nW10+KFtdKVxuY29uc3Qgb3B0aW9ucyA9IFtcbiAge1xuICAgIHZhbHVlOiAnSFRNTCcsXG4gICAgbGFiZWw6ICdIVE1MJyxcbiAgfSxcbiAge1xuICAgIHZhbHVlOiAnQ1NTJyxcbiAgICBsYWJlbDogJ0NTUycsXG4gIH0sXG4gIHtcbiAgICB2YWx1ZTogJ0phdmFTY3JpcHQnLFxuICAgIGxhYmVsOiAnSmF2YVNjcmlwdCcsXG4gIH0sXG5dXG5jb25zdCBpbnB1dE9wdGlvbiA9IHJlZignJylcbmZ1bmN0aW9uIGhhbmRsZUZpbHRlck1ldGhvZCh2YWx1ZSkge1xuICBpZih2YWx1ZSkgaW5wdXRPcHRpb24udmFsdWUgPSB2YWx1ZVxuICByZXR1cm4gdmFsdWVcbn1cbmZ1bmN0aW9uIGhhbmRsZVZpc2libGVDaGFuZ2UodmlzaWJsZSkge1xuICBpZighdmlzaWJsZSkge1xuICAgIC8vIOiuvue9rnNlbGVjdO+8jCDmuIXnqbppbnB1dE9wdGlvblxuICAgIGlucHV0T3B0aW9uLnZhbHVlICYmICF2YWx1ZS52YWx1ZS5pbmNsdWRlcyhpbnB1dE9wdGlvbi52YWx1ZSkgJiYgdmFsdWUudmFsdWUucHVzaChpbnB1dE9wdGlvbi52YWx1ZSlcbiAgICBpbnB1dE9wdGlvbi52YWx1ZSA9ICcnXG4gIH1cbn1cbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxoMT57eyBtc2cgfX08L2gxPlxuICA8ZWwtaW5wdXQgdi1tb2RlbD1cIm1zZ1wiIC8+XG5cbiAgPHA+XG4gICAgPGVsLWljb24gY29sb3I9XCJ2YXIoLS1lbC1jb2xvci1wcmltYXJ5KVwiPjxFbGVtZW50UGx1cyAvPjwvZWwtaWNvbj5cbiAgICBFbGVtZW50IFBsdXMge3sgZXBWZXJzaW9uIH19ICsgVnVlIHt7IHZ1ZVZlcnNpb24gfX1cbiAgPC9wPlxuICAgPGVsLXNlbGVjdFxuICAgIHYtbW9kZWw9XCJ2YWx1ZVwiXG4gICAgbXVsdGlwbGVcbiAgICBmaWx0ZXJhYmxlXG4gICAgYWxsb3ctY3JlYXRlXG4gICAgZGVmYXVsdC1maXJzdC1vcHRpb25cbiAgICA6ZmlsdGVyLW1ldGhvZD1cImhhbmRsZUZpbHRlck1ldGhvZFwiXG4gICAgOnJlc2VydmUta2V5d29yZD1cImZhbHNlXCJcbiAgICBwbGFjZWhvbGRlcj1cIkNob29zZSB0YWdzIGZvciB5b3VyIGFydGljbGVcIlxuICAgIEB2aXNpYmxlLWNoYW5nZT1cImhhbmRsZVZpc2libGVDaGFuZ2VcIlxuICA+XG4gICAgPGVsLW9wdGlvblxuICAgICAgdi1mb3I9XCJpdGVtIGluIG9wdGlvbnNcIlxuICAgICAgOmtleT1cIml0ZW0udmFsdWVcIlxuICAgICAgOmxhYmVsPVwiaXRlbS5sYWJlbFwiXG4gICAgICA6dmFsdWU9XCJpdGVtLnZhbHVlXCJcbiAgICAvPlxuICA8L2VsLXNlbGVjdD5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsInRzY29uZmlnLmpzb24iOiJ7XG4gIFwiY29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcImFsbG93SnNcIjogdHJ1ZSxcbiAgICBcImNoZWNrSnNcIjogdHJ1ZSxcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVwiOiBcIkVTTmV4dFwiLFxuICAgIFwibW9kdWxlUmVzb2x1dGlvblwiOiBcIkJ1bmRsZXJcIixcbiAgICBcImFsbG93SW1wb3J0aW5nVHNFeHRlbnNpb25zXCI6IHRydWVcbiAgfSxcbiAgXCJ2dWVDb21waWxlck9wdGlvbnNcIjoge1xuICAgIFwidGFyZ2V0XCI6IDMuM1xuICB9XG59XG4iLCJfbyI6e319)
